go/registry/api/runtime.go

```diff
@@ -225,6 +225,24 @@ type EntityWhitelistConfig struct {
 type RuntimeAdmissionPolicy struct {
 	AnyNode         *AnyNodeRuntimeAdmissionPolicy         `json:"any_node,omitempty"`
 	EntityWhitelist *EntityWhitelistRuntimeAdmissionPolicy `json:"entity_whitelist,omitempty"`
+
+	// PerRole is a per-role admission policy that must be satisfied in addition to the global
+	// admission policy for a specific role.
+	PerRole map[node.RolesMask]PerRoleAdmissionPolicy `json:"per_role,omitempty"`
```

Comment:
Not sure what we gain with `role -> entity -> maxNodes` policy as we already have `entity -> role -> maxNodes` policy.

Reply:
Imagine where you only want to enforce a whitelist for a specific role, but not any others.